compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUTileAndFuse.cpp

```diff
@@ -209,19 +209,24 @@ void LLVMCPUTileAndFusePass::runOnOperation() {
   }
```

Comment:
Is it possible to move all the `maybeLoweringConfig` code inside the `else` branch below? We shouldn't execute it if we use the consumer lowering config?